ci.yml

```diff
@@ -0,0 +1,4 @@
+deps:
+- name: pmm-update
+  branch: PMM-11126
+  url: https://github.com/percona/pmm-update
```

Comment:
You can as well skip the `url` it'll work without it :)

Reply:
@atymchuk this is what `make prepare XXX` does :)